src/wms/src/test/java/org/geoserver/wms/wms_1_3/CapabilitiesSystemTest.java

```diff
@@ -334,6 +334,12 @@ public void testCiteCompliant() throws Exception {
 
             dom = getAsDOM("wms?request=GetCapabilities&version=1.3.0");
             checkLegacyException(dom, ServiceException.MISSING_PARAMETER_VALUE, "service");
+
+            dom = getAsDOM("wcs/wms?request=GetCapabilities&version=1.3.0");
+            checkLegacyException(dom, ServiceException.MISSING_PARAMETER_VALUE, "service");
+
+            dom = getAsDOM("wcs/wms/?request=GetCapabilities&version=1.3.0");
+            checkLegacyException(dom, ServiceException.MISSING_PARAMETER_VALUE, "service");
```

Comment:
I'm confused about what this is supposed to do.... wcs seems to take the role of a workspace here?

Reply:
Yeps, the setupData adds a workspace names `wcs`

Reply:
Aaah... ok, I guess it would help to add a comment specifying that "wcs" is a workspace there.

Reply:
Done